### DISCUSSION TITLE
5.1.5 Release

### DISCUSSION THREAD

**Discussion by marr** [lerna/lerna#3196] [Q&A] [answered]
Hi,
Looking forward to the next release. I am upgrading from v4, and am very much looking forward to the `package-lock.json` whitespace issue being gone.

Any idea when that will be released?

Thanks!
Dave

**Comment by JamesHenry** [comment, marked as answer]
Hi @marr https://github.com/lerna/lerna/releases/tag/v5.1.5 is now available
